.eslintrc.json

```diff
@@ -4,32 +4,15 @@
     "mocha": true
   },
   "extends": [
-    "@1stg",
-    "plugin:eslint-plugin/recommended"
```

Comment:
A lot of plugins are missing in this PR.

Reply:
And also there some special logic inside

1. https://github.com/1stG/configs/blob/master/packages/config/index.js
2. https://github.com/1stG/configs/blob/master/packages/lint-staged/base.js#L7-L8

Reply:
It's hundreds line project! What for ?

Reply:
That's why I'm using presets here.